engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -282,23 +271,34 @@ class EngineJobExecutionActor(replyTo: ActorRef,
     *
     * @return a RoundRobinPool of actors based on this.factory.
     */
-  private def fileContentsHasherActor: ActorRef = {
+  private[execution] lazy val fileContentsHasherActor: ActorRef = {
```

Comment:
Revert this. It's busted as the actors will die when the EJEA dies. I've fixed it in #1411.